gdpr/management/commands/deactivate_expired_reasons.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+from gdpr.models import LegalReason
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        expired_reasons_retaining_data = LegalReason.objects.filter_expired_retaining_data_in_last_days()
+
+        for legal_reason in expired_reasons_retaining_data:
+            if not legal_reason.source_object_has_active_non_expired_legal_reasons():
```

Comment:
nemělo by to být per data? Každý reason může mít jiné data co by se měly anonymizovat